pkg/kdump/kdump-view.jsx

```diff
@@ -494,47 +494,54 @@ export class KdumpPage extends React.Component {
         return (
             <Page>
                 <PageSection variant={PageSectionVariants.light}>
-                    <Flex justifyContent={{ default: "justifyContentCenter" }}>
-                        <DescriptionList>
-                            <DescriptionListGroup>
-                                <DescriptionListTerm>{_("kdump status")}</DescriptionListTerm>
-                                <DescriptionListDescription>
-                                    <div role="group">
-                                        <Switch isChecked={!!serviceRunning}
-                                                onChange={this.props.onSetServiceState}
-                                                aria-label={_("kdump status")}
-                                                isDisabled={this.props.stateChanging} />
+                    <span className="kdump-heading-title-group">
```

Comment:
Why you dropped `Flex` and instead wrote custom css? Is there something which `Flex` cannot do and we need our own css?

Reply:
I just copied the CSS from the firewall.css, I think I can do it from React as well. I can repost or send a followup and change it in the other pages as well, aka firewall, selinux, they all use the same flex for the headers.

Reply:
Ah right, I see. Your call :) 
I would prefer slightly more doing it here as this PR introduces unnecessary css but I am fine with followup as well. I'll give you ack not to block this.

Reply:
This PR, it's not really fixing anything, it's just redesign. So I am not in a hurry to merge it. I agree on your point on not introducing custom CSS, I am shooting at my own leg here basically :package: 

I will work on this PR. Moving it to draft.